### PR TITLE
Added list of commands with short descriptions to USAGE.md

### PR DESCRIPTION
**Pull request by anfly0**
Not 100% verified, but close to it.
Should at least serve as a good start.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -3,4 +3,118 @@
...
+>	- __A__ - Move cursor to the end of the current row and enter insert mode .
+>	- __HOME__ - Move cursor to the start of the current row.
+>	- __END__ - Move cursor to the end of the current row.
+>	- __PAGE UP__ - Move cursor to the first row.
+>	- __PAGE DOWN__ - Move cursor to the last line or move cursor to the last visual line (??).
+>	- __ctrl + m__ - Jump to symbol definition .
+>	- __ctrl + n__ - Jump back to previous position after __ctrl + m__ .
+
+2. ### Edit
+>	- __rc__ - Copy current row.
```
Comment (shnupta) [inline USAGE.md]: The following should be reversed. e.g. `cr` instead of `rc`

File: USAGE.md [hunk 2]
```diff
@@ -3,4 +3,118 @@
...
+
+## Command mode
+
+1. ### Movement
+>	- __*[0-9]__ - Move cursor to specified line number.
+
+2. ### Misc
+>	- __q!__ - Force quit without saving changes.
+>	- __w__ - Save changes.
+>	- __wp__ - Save changes and quit.
```
Comment (shnupta) [inline USAGE.md]: typo `wp` should be `wq`

File: USAGE.md [hunk 3]
```diff
@@ -3,4 +3,118 @@
...
+
+2. ### Edit
+>	- __rc__ - Copy current row.
+>	- __ry__ - Cut current row.
+>	- __rp__ - Paste row from editor buffer.
+>	- __rd__ - Delete current row.
+>	- __pc__ - Paste from clipboard.
+
+3. ### Misc
+>	- __cc__ - ???.
```
Comment (shnupta) [inline USAGE.md]: I don't believe there is a `cc` command at the moment

**Comment by shnupta** [comment]
Hey, nice start. We can keep adding to this.

We should also probably add links to each section at the beginning of the file.

**Comment by anfly0** [comment]
I've fixed the typos and and added some rudimentary anchor links.
But i'm still wondering about the cc command in Normal mode, this is a part of the "Normal mode" switch statement: 
https://github.com/shnupta/bric/blob/2ed07f1a3f679298e742a0f4a553e7982b75f572/bric.c#L2026-L2028

**Comment by anfly0** [comment]
I've fixed the typos and added some rudimentary anchor links.
But i'm still wondering about the 'cc' command in Normal mode, this is a part of the "Normal mode" switch statement : 
https://github.com/shnupta/bric/blob/2ed07f1a3f679298e742a0f4a553e7982b75f572/bric.c#L2026-L2028

**Comment by shnupta** [comment]
Ohhh yes sorry my bad. That's to clear whatever is in the clipboard.

**Pull request merged**